bittide-instances/src/Bittide/Instances/Hitl/SwCcTopologies.hs

```diff
@@ -196,6 +183,8 @@ data TestConfig = TestConfig
   -- cycles of the stable clock.
   , mask :: BitVector LinkCount
   -- ^ The link mask depending on the selected topology.
+  , reframingEnabled :: Bool
+  -- ^ Whether or not to run this test with reframing enabled.
```

Comment:
Create an issue we need to figure out how to properly test reframing and link it here.

Reply:
Made #693, will link in the comment.